js/repl/compile.js

```diff
@@ -81,7 +81,7 @@ export default function compile(code: string, config: CompileConfig): Return {
   try {
     const babelConfig = {
       babelrc: false,
-      filename: "repl",
+      filename: config.fileName,
```

Comment:
I think I'd prefer we name this `config.filename` in lowercase so it would be more consistent with Babel's own option name.

Reply:
On the sample page, this appears to not be set. Not sure what the issue is. Maybe it's not getting passed along somewhere?